pkg/cmd/pipelinerun/pipelinerun.go

```diff
@@ -26,6 +26,7 @@ import (
 func Command(p cli.Params) *cobra.Command {
 	c := &cobra.Command{
 		Use:     "pipelineruns",
+		DisableFlagsInUseLine: true,
```

Comment:
Why is it required ? :thinking:

Reply:
To disable the additional flag note in the usage of `tkn pr -h`.
It doesn't help the user

Reply:
FYI: There are 2 other settings that I want to add to `root` command 
`SilenceUsage, SilenceErrors`  so that the usage isn't displayed unless there is good reason to.